calm-business/sass/site/header/_site-header.scss

```diff
@@ -71,9 +71,7 @@
 
 // Site title
 
-.site-title,
-.site-title.wp-block-a8c-site-title,
-.site-title.wp-block-a8c-site-title:focus {
```

Comment:
this looks like the scss change is not correct, the compiled css on svn does have these rules: https://github.com/Automattic/themes/blob/71e1225672cd60f8143f77411db2475e2306a2bb/calm-business/style.css#L2010